css/partials/_layout.scss

```diff
@@ -263,4 +278,9 @@ html.no-js .js-hidden {
 	@include bp-tablet--portrait {
 		display: none;
 	}
+}
+
+.content {
+    margin-top: 36px;
```

Comment:
When .content comes after .title-page the spacing is too big. Can you add a new rule to remove the top margin when they follow one another?